Snippets/Snippets_6/Callback/Usage.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Snippets6.Callback
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+
+    public class Usage
+    {
+        public async Task Describe()
+        {
+            IBus bus = null;
+            var options = new SendOptions();
+
+            #region CallbackWithMessageAsResponse
+            var responseMessage = await bus.RequestWithTransientlyHandledResponseAsync<PlaceOrderResponse>(new PlaceOrder(), options);
```

Comment:
Good question. Might be overkill if most APIs become async

Am Freitag, 17. Juli 2015 schrieb Simon Cropp :

> In Snippets/Snippets_6/Callback/Usage.cs
> <https://github.com/Particular/docs.particular.net/pull/708#discussion_r34911650>
> :
>
> > @@ -0,0 +1,32 @@
> > +﻿namespace Snippets6.Callback
> > +{
> > +    using System;
> > +    using System.Threading.Tasks;
> > +    using NServiceBus;
> > +
> > +    public class Usage
> > +    {
> > +        public async Task Describe()
> > +        {
> > +            IBus bus = null;
> > +            var options = new SendOptions();
> > +
> > +            #region CallbackWithMessageAsResponse
> > +            var responseMessage = await bus.RequestWithTransientlyHandledResponseAsync<PlaceOrderResponse>(new PlaceOrder(), options);
>
> are we planning on suffix all async member in NSB V6 with Async?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/708/files#r34911650>
> .
>